pynamodb/models.py

```diff
@@ -250,6 +251,16 @@ def __init__(self, hash_key=None, range_key=None, **attributes):
             attributes[self._range_keyname] = range_key
         super(Model, self).__init__(**attributes)
 
+    def get_hash_key(self):
```

Comment:
@jpinner-lyft is planning to introduce
```py
    @property
    def _hash_key(self):
        return getattr(self, self._hash_keyname)

    @property
    def _range_key(self):
        return getattr(self, self._range_keyname) if self._range_keyname else None
```
in this #644.

Maybe you should introduce the same. Note that his version introduces private methods: are you OK with private methods?

Reply:
Currently I'm accessing these within the `Transaction` object, so they wouldn't be used privately :0

Reply:
im actually not using these anymore so im deleting them from my pr